### PR TITLE
Initalize documentation file struct and build script for chinese language .

### PR DESCRIPTION
**Pull request by lugreen**
Initalize documentation file struct and build script  for chinese language .

### CODE REVIEW

**Comment by lugreen** [comment]
Initalize documentation file struct and build script for chinese language .

File: build.gradle [hunk 1]
```diff
@@ -57,6 +62,12 @@ task chopManualRu(dependsOn: [buildManualRu], type: CreateMultiPageDoc) {
     docTitle = "<a href='https://www.cuba-platform.ru/manual' target='_blank'>Платформа CUBA</a>. Руководство по разработке приложений"
 }
 
+task chopManualCn(dependsOn: [buildManualEn], type: CreateMultiPageDoc) {
```
Comment (oshiryaeva) [inline build.gradle]: task `chopManualCn` should depend on `buildManualCn`, not En.

File: build.gradle [hunk 2]
```diff
@@ -67,6 +78,11 @@ task warManualRu(dependsOn: [chopManualRu], type: CreateWar) {
     docLang = 'ru'
 }
 
+task warManualCn(dependsOn: [chopManualEn], type: CreateWar) {
```
Comment (oshiryaeva) [inline build.gradle]: task `warManualCn` should depend on `chopManualCn`, not En.

File: build.gradle (hunk 1 above)
Comment (lugreen) [reply build.gradle]: Hi,thanks.I have fixed that.

File: build.gradle (hunk 2 above)
Comment (lugreen) [reply build.gradle]: Hi,thanks.I have fixed that.

**Pull request merged**